docs/html/development/getting-started.rst

```diff
@@ -34,12 +34,16 @@ For developing pip, you need to install :pypi:`tox`. Often, you can run
 Running pip From Source Tree
 ============================
 
-To run the pip executable from your source tree during development, run pip
-from the ``src`` directory:
+To run the pip executable from your source tree during development, install pip
+locally using editable installation (inside a virtualenv).
+You can then invoke your local source tree pip normally.
 
 .. code-block:: console
 
-    $ python src/pip --version
+    $ virtualenv venv # You can also use "python -m venv venv" from python3.3+
+    $ source venv/bin/activate
+    $ python -m pip install -e .
+    $ python -m pip --version
```

Comment:
IIUC, we can do `pip install -e .` and `pip --version` if we are inside a virtualenv as well, since we know that `pip` inside the virtualenv is pointing to our source tree, so using `python -m pip` is equivalent to using `pip`.

But sticking with `python -m pip` will [also be a good idea](https://snarky.ca/why-you-should-use-python-m-pip/)

Reply:
I think we should prefer teaching and using `python -m pip` in any case as a good practice.

Reply:
I'm very inclined to opposed that -- but I know better than to pick up the wrong topics in the wrong place. :)